blueprints/twiddle.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "jquery": "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.js",
     "ember": "https://cdnjs.cloudflare.com/ajax/libs/ember.js/1.13.5/ember.js",
```

Comment:
Apparently it takes a little while to sync with CDN.js here.  We can leave this as 1.13.5 for now (and update once they have propagated).